lib/outputs/outputs.py

```diff
@@ -252,3 +254,29 @@ def close_json(output_directory, audited_account, mode):
     file_descriptor.truncate()
     file_descriptor.write("]")
     file_descriptor.close()
+
+
+def send_to_s3_bucket(output_directory, output_mode, output_bucket, audit_info):
```

Comment:
Please, include tests for this function.

Reply:
Sure